### ISSUE TITLE
Inference problem

### ISSUE THREAD

**Issue by SoYeol-Lee** [Stick-To/RefineDet-tensorflow#12]
Thank you for your share!

When I run refence code in test.py, I got the follow result:

![result](https://user-images.githubusercontent.com/59550240/86015188-36f4ec80-ba5c-11ea-9485-6f44a3aed675.png)

Training loss were reduced well.

![fig1](https://user-images.githubusercontent.com/59550240/86015560-b1be0780-ba5c-11ea-8201-bcfb6717c465.png)

My config setting is 

```
config = {
    'mode': 'train',                            
    'input_size': 320,                         
    'data_format': 'channels_last',             
    'num_classes': 20,
    'weight_decay': 1e-4,
    'keep_prob': 0.5,                           
    'batch_size': batch_size,
    'nms_score_threshold': 0.45,
    'nms_max_boxes': 20,
    'nms_iou_threshold': 0.45,
    'pretraining_weight': ckpt_path
}
```

How can I fix this problem? or, has anyone got the right result by this code?

**Comment by tmlstrong** [comment]
Hi @SoYeol-Lee, I have same issue. How did you fix it? Thank you for share!
